client/app/app.js

```diff
@@ -0,0 +1,10 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+var WikiPoetryApp = require('./components/WikiPoetryApp.react')
+
+
+// Render FluxCartApp Controller View
+ReactDOM.render(
+  <WikiPoetryApp />,
+  document.getElementById('flux-wiki')
```

Comment:
Instead of having this getElementById, I think we can have line 8 be <WikiPoetryApp, flux-wiki />. It's something to do with ids being put in the global scope when they are defined.